uds/diagnostic_configuration/state.py

```
@@ -0,0 +1,85 @@
+"""Implementation for diagnostic communication state."""
+
+__all__ = ["State"]
+
+from typing import Any, Collection, Optional, Set
```

Ruff (0.15.20)

[warning] Import from `collections.abc` instead: `Collection`

Import from `collections.abc`

(UP035)